EiDiA_frontend/src/views/FileCabinetView.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+import React from 'react';
+import Page from "../components/Page";
+import {RecordSymbol} from "../components/record/RecordSymbol";
+import styled from "styled-components";
+import {Input} from '@material-ui/core';
+import {Link} from "../components/Link";
+
+const FlexRow = styled.div`
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    flex-wrap: wrap;
+`;
+
+const SearchBar = styled.div`
+    width: 80%;
+    justify-content: center;
+    margin: 3% 10%;
+`;
+
+export class FileCabinetView extends React.Component {
+
+    /*
+     *TODO:
+     * - add getRecordFromDatabase functionality
+     * - add Pages if too many records
+     * - add Add Button
+     */
+    constructor(props) {
+        super(props);
+        this.state = {
+            records: ["Volkswagen",
+                "BMW",
+                "Thyssenkrup",
+                "Google",
+                "Facebook",
+                "Microsoft",
+                "ABC Company",
+                "Adidas",
+                "lenovo Limited",
+                "IBM",
+                "TrueThat",
+                "hello",
+                "abc",
+                "def"
+            ],
+            search: '',
+        }
+    }
+
+    componentDidMount() {
+        this.props.setTitle("File Cabinet");
+    }
+
+    updateSearch(event) {
+        this.setState({search: event.target.value.substr(0, 20)});
+    }
+
+    render() {
+        let filteredRecords = this.state.records.filter((record) => {
+            return record.indexOf(this.state.search) !== -1;
+        });
+
+        return (
+            <Page title={"File Cabinet"}>
+                <SearchBar>
+                    <Input
+                        placeholder="Search Records ..."
+                        fullWidth={true}
+                        inputProps={{'aria-label': 'description'}}
+                        value={this.state.search}
+                        onChange={this.updateSearch.bind(this)}/>
+                </SearchBar>
+
+
+                <FlexRow>
+                    {/*
+                    TODO: make Link depending on the records
+                    */}
+                    {filteredRecords.map((record, i, records) =>
+                        <Link
+                            key={records[i]}
+                            to={"/record"}>
```

Comment:
```suggestion
                    {filteredRecords.map((record, i) =>
                        <Link
                            key={i}
                            to={"/record/" + record.id}>
```
damit man zum richtigen record linkt